server/app/services/settings/SettingsManifest.java

```diff
@@ -972,6 +977,12 @@ public boolean getProgramFilteringEnabled(RequestHeader request) {
                       /* isRequired= */ true,
                       SettingType.STRING,
                       SettingMode.ADMIN_WRITEABLE),
+                  SettingDescription.create(
+                      "CIVIC_ENTITY_PRODUCTION_URL",
+                      "The URL to the civic entity's production CiviForm site.",
```

Comment:
**suggestion:** I think its worth adding another sentence to explain what this is used for like: "Providing this on a staging site will add a banner indicating applicants shouldn't submit real applications, but instead submit them at the link provided."